lib/codemod/src/transforms/__testfixtures__/update-organisation-name/update-organisation-name.input.js

```diff
@@ -1,7 +1,6 @@
 /* eslint-disable */
 import '@kadira/storybook-addons';
 import '@kadira/react-storybook-decorator-centered';
-import '@kadira/storybook-addon-comments';
```

Comment:
What if a project actually has this dependency? Currently it's replaced with `@storybook/react-addon-comments`, which is a non-existent package

Reply:
See also: https://github.com/storybooks/storybook/commit/cf054ad1b35a7e934c18e10db411bff8d39aaa8d#commitcomment-25450913

Reply:
Good point 😄

Reply:
@Hypnosphi @danielduan any idea how this passed in https://github.com/storybooks/storybook/pull/2189 

I'm not so familiar with how our codemod works?

Reply:
#2189 doesn't change anything in codemod

Reply:
Updated codemod & test cases per your suggestions @Hypnosphi . Also ran `npm deprecate`